src/client/src/components/pages/dashboard/index.jsx

```diff
@@ -11,7 +12,9 @@ class index extends Component {
   state = {
     participant: "",
     course: "",
-    worker: ""
+    worker: "",
+    employed: "",
```

Comment:
instead of  store employed and UnEmployed you can store sections in state like : 

```
sections: [
{ title: 'Employed', percentage: 0 },
{ title: 'Unemployed', percentage: 0 }
]
```

Reply:
Its really can be more useful
I do it